frontend/src/static/js/v_summary.js

```diff
@@ -803,10 +803,9 @@ window.vSummary = {
 
     restoreZoomFiltered(info) {
       const {
-        zSince, zUntil, zFilterGroup, zTimeFrame, zIsMerge, zSorting, zSortingWithin,
-        zIsSortingDsc, zIsSortingWithinDsc,
+        zSince, zUntil, zTimeFrame, zIsMerge,
       } = info;
-      let filtered = [];
+      const filtered = [];
```

Comment:
Can you just leave a comment here as to why the shift from `let` to `const`?

Reply:
it will not be reassigned so by definition const would be better classifier. Actually this is the result of js auto correction